pallets/eth-bridge/src/benchmarking.rs

```diff
@@ -207,7 +206,7 @@ benchmarks! {
     }: _(RawOrigin::None, tx_id, new_confirmation.clone(), author.clone(), signature)
     verify {
         let active_tx = ActiveRequest::<T>::get().expect("is active");
-        ensure!(active_tx.confirmations.contains(&new_confirmation), "Confirmation not added");
+        ensure!(active_tx.confirmation.confirmations.contains(&new_confirmation), "Confirmation not added");
```

Comment:
`confirmation.confirmations` ?

Reply:
Confirmation is a property that contains confirmation related data such as the message hash and the list of confirmations. I could call it `confimration_data` but that was annoying me because its longer :)